browser/extensions/api/settings_private/brave_prefs_util.cc

```diff
@@ -32,6 +33,10 @@
 #include "brave/components/brave_wallet/browser/pref_names.h"
```

Comment:
do we still need this and associated deps?

Reply:
Yes we still need it because `kBraveWalletWeb3Provider` is not going to be deprecated